### PR TITLE
Bwv 528 update

### PR DESCRIPTION
**Pull request by metzguar**
Corrected a few notes in 1st and 2nd movement.
Kept old Lilypond version 2.8.4, as newer versions (tried 2.14 and 2.18)
didn't produce correct beaming of triplets in 3rd movement.

### CODE REVIEW

**Comment by chrissawer** [comment]
Can anyone confirm these note corrections before I merge?
Lilypond 2.8 should be fine, it is the oldest version we have installed on the build server.

File: ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Adagio.ly [hunk 1]
```diff
@@ -21,7 +21,7 @@ bintro = \relative c
 {
 	e4 r8 dis e4 r8 fis|
 	g4 r8 c8 fis,4 r8 h,|
-	e, e' g h h, c'16 h c8 c,|
```
Comment (glenl) [inline ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Adagio.ly]: +1

File: ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Adagio.ly [hunk 2]
```diff
@@ -214,8 +214,8 @@ bassA = \relative c
 	e d c|
 %35
 	h4 r8 h e h|
-	c4 r8 f d d|
```
Comment (glenl) [inline ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Adagio.ly]: +1

File: ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Adagio.ly [hunk 3]
```diff
@@ -243,7 +243,7 @@ bassA = \relative c
 	h h' fis a dis, fis|
 %59
 	a,4 r8 c h a|
-	g a h g h4|
```
Comment (glenl) [inline ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Adagio.ly]: +1

File: ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Andante.ly [hunk 4]
```diff
@@ -54,7 +54,7 @@ sopranB = \relative c'
 	g16 a,32 h cis h a16 d a e' a, fis'2~ |
 %38
 	fis16 g,32 a h g fis g cis g fis g d' g, fis g
-		e'16 g, fis d' fis, d' cis fis, |
```
Comment (glenl) [inline ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Andante.ly]: +1

File: ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Andante.ly [hunk 5]
```diff
@@ -114,7 +114,7 @@ altB = \relative c'
 	g, e h'8 r4 r16 fis'32 e d fis g fis cis fis g fis h, fis' g fis |
 	ais,8 h ais gis fis32 ais h cis d h fis d'cis ais fis cis'h gis eis h'|
 %35
-	ais8 h ais gis fis16 fis32 gis a16 fis h fis cis' fis, |
```
Comment (glenl) [inline ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Andante.ly]: +1

File: ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Andante.ly [hunk 6]
```diff
@@ -142,7 +142,7 @@ bassB = \relative c'
 %8
 	d r h r cis fis cis' cis, |
 	fis r d r e r cis r |
-	d r h r cis fis cis cis |
```
Comment (glenl) [inline ftp/BachJS/BWV528/SonataIV/SonataIV-lys/Andante.ly]: +1

**Pull request merged**